charts/zookeeper-operator/README.md

```diff
@@ -53,3 +53,7 @@ The following table lists the configurable parameters of the zookeeper-operator
 | `nodeSelector` | Map of key-value pairs to be present as labels in the node in which the pod should run | `{}` |
 | `affinity` | Specifies scheduling constraints on pods | `{}` |
 | `tolerations` | Specifies the pod's tolerations | `[]` |
+| `additionalSidecars:logging.enable` | If true, operator logs forward to Splunk | `true` |
```

Comment:
No need for this flag

Reply:
Removed additionalSidecars:logging.enable